src/common.h

```diff
@@ -129,11 +129,12 @@ int ishexlcn(const char *q, int n);
 size_t hexlen(const char *q, int *extra_chars);
 size_t hexlenl(const char *q, int *extra_chars); /* lower cased only */
 size_t hexlenu(const char *q, int *extra_chars); /* upper cased only */
-/* is this a valid string for atoi() ONLY positive numbers are valid */
+/*Is this a valid number.
+ *ONLY positive numbers are valid. */
 int isdec(const char *q);
-/* is this a valid string for atoi() */
+/*Is this a valid number */
 int isdec_negok(const char *q);
-/* is this a valid string for atou()?  atou() func == sprintf("%x",&val) */
+/* Is this a valid number*/
```

Comment:
These comments are non-descriptive (same comment for both functions, then what's the difference between them?), and besides the (lack of) whitespace before/after asterisks is sloppy.